packages/superset-ui-connection/src/callApi/callApi.ts

```diff
@@ -16,6 +21,12 @@ export default function callApi({
   stringify = true,
   url,
```

Comment:
Can we add the retry options to `callApi` arguments, too, in case people want to override the defaults?

It'd also be preferred to make this an opt-in rather than opt-out. Can also add an API to `superset-ui/connection` to change the global defaults.

Reply:
I've added the retry options to the SupersetClient config. However, I think it's fair to make this opt out. I can add a comment in incubator-superset that this was added, but I think the common path should include QOL features like automatic retried on network issues